polling_stations/apps/data_importers/management/commands/import_cheshire_west.py

```diff
@@ -20,45 +20,38 @@ def address_record_to_dict(self, record):
             "10091137622",  # 72 MUSKETT DRIVE, NORTHWICH
             "10091137631",  # 68 MUSKETT DRIVE, NORTHWICH
             "200002842851",  # GREVILLE HOUSE, GREVILLE DRIVE, WINSFORD
-            "100012372274",  # STANTHORNE HALL FARM, MIDDLEWICH ROAD, STANTHORNE, MIDDLEWICH
-            "100012353201",  # STREET FARM, KELSALL ROAD, TARVIN, CHESTER
             "200000831359",  # OAK BANK, PARKGATE ROAD, CHESTER
-            "200000832902",  # LOWCROSS MILL, LOWCROSS, TILSTON, MALPAS
-            "200000990850",  # KIDNAL COTTAGE, KIDNAL, MALPAS
+            "200000990849",  # KIDNAL HOUSE, KIDNAL, MALPAS
+            "100012360649",  # 6 LEIGHTON COTTAGES, BOATHOUSE LANE, PARKGATE, NESTON
+            "10095485945",  # WESTVIEW, WELSH ROAD, LEDSHAM, ELLESMERE PORT
+            "10094685291",  # 6 BALISTER DRIVE, HARTFORD, NORTHWICH
         ]:
             return None
 
         if record.addressline6 in [
             # splits
-            "CW7 3EQ",
-            "CW9 8PU",
-            "CH66 1NZ",
             "CW9 6EL",
             "CW9 8XB",
-            "CH65 9JU",
-            "CW8 4AB",
+            "CW9 7SQ",
+            "CH66 1NZ",
+            "CW9 8PU",
             "CW7 2GG",
-            "CW9 7RX",
             "CH64 3SG",
-            "CW8 2NQ",
+            "CW7 3EQ",
+            "CH4 7DG",
+            "CW8 4AB",
             "CW6 9EP",
+            "CW9 7RX",
+            "CW8 4QS",
+            "CH65 9JU",
             "WA6 0JA",
+            "WA6 6LN",
         ]:
             return None
 
         return super().address_record_to_dict(record)
 
     def station_record_to_dict(self, record):
-        # The Upton Royal British Legion, 20 Heath Road, Upton, Chester, CH1 1HX
-        # Proposed postcode correction: CH2 1HX
-        if record.polling_place_id == "9689":
-            record = record._replace(polling_place_postcode="")
-
-        # Mobile Polling Station - Cheviot Square, Cheviot Square, Winsford
-        # Proposed postcode correction: CW7 1QS
-        if record.polling_place_id == "9643":
-            record = record._replace(polling_place_postcode="")
-
         # Lower Peover CE Primary School, The Cobbles, Lower Peover, Knutsford, Cheshire, WA16 9PZ
         # Fixing the warning, correction is cosmetic and safe
         if record.polling_place_id == "9447":
```

Comment:
I'm surprised the id hasn't changed. Are you sure?

Reply:
You are right!